.github/workflows/ci.yaml

```diff
@@ -75,3 +75,39 @@ jobs:
           file: coverage.xml
           fail_ci_if_error: False
           verbose: True
+
+  package-tests:
```

Comment:
Thoughts on splitting this into it's own yaml? If the `package-tests` job only runs on schedule, then we can just make something like `package-tests.yaml`

Reply:
Can do, I believe the other job also runs on schedule, but I do see the appeal of a cleaner "independent jobs per yaml" approach.